src/components/List/List.js

```diff
@@ -46,39 +41,44 @@ const List = ({
   verticalAlign,
   horizontalAlign,
   ...otherProps
-}) => {
-  return (
-    <StyledList
-      inline={inline}
-      verticalAlign={verticalAlign}
-      horizontalAlign={horizontalAlign}
-      shadow={shadow}
-      fullWidth={fullWidth}
-      className={className}
-      style={style}
-      {...otherProps}
-    >
-      {children}
-    </StyledList>
-  );
-};
+}) => (
+  <StyledList
+    inline={inline}
+    verticalAlign={verticalAlign}
+    horizontalAlign={horizontalAlign}
+    shadow={shadow}
+    fullWidth={fullWidth}
+    className={className}
+    style={style}
+    {...otherProps}
+  >
+    {children}
+  </StyledList>
+);
 
 List.defaultProps = {
   style: {},
   fullWidth: false,
   shadow: true,
-  inline: false
+  inline: false,
+  className: '',
+  children: null,
+  verticalAlign: 'bottom',
```

Comment:
Let's remove default values for `verticalAlign` and `horizontalAlign` mate 🎉 it was left out on purpose

Reply:
ok, fixed!